lib/metadata/util/md5deep.rb

```diff
@@ -47,30 +48,48 @@ def scan(path, rootID = "/")
   end
 
   def self.scan_glob(fs, filename, options = {})
+    begin
     md5 = MD5deep.new(fs, options)
     md5.scan_glob(filename)
+    rescue Exception => err
+      $log.info "MD5.scan_glob: Exception #{err} rescued"
+      $log.debug err.backtrace.join("\n")
+    end
   end
 
   def scan_glob(filename)
     filename.tr!("\\", "/")
     startDir = File.dirname(filename)
     globPattern = File.basename(filename)
```

Comment:
Is this still used with the new approach?

Reply:
@chessbyte if you mean the globPattern variable, yes.

Reply:
@jerryk55 it seems to be only used in the true leg of `if @fs.fileExists?(filename)` implying that it is not a glob pattern inside, but, rather, something else.  Suggest the `File.basename(filename)` call be moved inside the if statement and assigned to a more descriptive variable name.

Reply:
@chessbyte agreed.  processFile does not treat it as a glob pattern - its really the basename of the file.  I'll move and rename it.

Reply:
@chessbyte agreed.  processFile does not treat it as a glob pattern - its really the basename of the file.  I'll move and rename it.